src/app/containers/StoryPromo/LinkContents/index.jsx

```diff
@@ -13,7 +13,7 @@ const LinkContents = ({ item, isInline }) => {
     translations: { media: mediaTranslations },
   } = useContext(ServiceContext);
 
-  const isMedia = pathOr(null, ['cpsType'], item) === 'MAP';
+  const isMedia = !!pathOr(null, ['media'], item);
```

Comment:
Is this a bit more readable?
```suggestion
  const isMedia = pathOr(false, ['media'], item);
```

Reply:
These are not the same; One is checking for the true/false association, the other returns the value at that path. You can use hasPath to return true/false.